src/PackagePrivate/AddNormalizedTermsTablesDDL.sql

```diff
@@ -0,0 +1,110 @@
+-- General info and links:
+-- -----------------------
+--
+-- These tables represent a normalized form of the wb_terms table.
+--
+-- The wb_terms table history leading to this point can be all followed starting
+-- from this epic https://phabricator.wikimedia.org/T208425
+--
+-- The current solution is part of that epic. work on it can be followed starting
+-- from this ticket https://phabricator.wikimedia.org/T219175
+--
+-- This design is normalized for the sake of reducing duplication and redundancy
+-- that accounted for ~70% of the disk space occupied by wb_terms table as of
+-- March 2019. It requires few joins for queris, and impact on performance
+-- will be addressed as the need arise.
+--
+-- Note on splitting Items and Properties:
+-- ---------------------------------------
+-- wb_item_terms an wb_property_terms are two tables with similar structure.
+-- original draft had them combined in one table, but further discussion lead
+-- to the decision of splitting them for the following reasons:
+-- - items are lot more and will grow faster than properties, which means
+--   when querying only for properties we need not to look at a gigantic
+--   index that includes items as well, but rather a very small one.
+-- - conceptually, items and properties are two different entity types, and
+--   since different entity types might have different constraints and maybe
+--   even structure, splitting them into corrisponding tables saves us from
+--   ending up with a wide polymorphic table again (lesson learned from
+--   wb_terms table itself)
+--
+-- Note on Entity IDs:
+-- ------------------
+-- Since wikidata entity ids are fixed and known to always be Q (like in Q123),
```

Comment:
This sentence uses the term "entity" in a confusing way. For what this codebase currently cares about, only Item IDs and Property IDs are known to be prefixed integers. Saying that (all) "wikidata entity ids are fixed" can be very confusing. It would be good to avoid the terminology "Entity ID" here.